store/sqlstore/channel_store.go

```diff
@@ -3442,6 +3442,39 @@ func (s SqlChannelStore) UserBelongsToChannels(userId string, channelIds []strin
 	return c > 0, nil
 }
 
+func (s SqlChannelStore) GetDirectChannelsForUser(userId string) ([]*model.Channel, *model.AppError) {
```

Comment:
Let's return `error` instead of `*model.AppError`. Take a look at some other methods of the store.

Reply:
done